quesma/model/transformation_pipeline.go

```diff
@@ -0,0 +1,26 @@
+// Copyright Quesma, licensed under the Elastic License 2.0.
+// SPDX-License-Identifier: Elastic-2.0
+package model
+
+type TransformationPipeline struct {
+	transformers []QueryTransformer
+}
+
+func NewTransformationPipeline() *TransformationPipeline {
+	return &TransformationPipeline{}
+}
+
+func (o *TransformationPipeline) Transform(queries []*Query) ([]*Query, error) {
+	var err error
+	for _, transformer := range o.transformers {
+		queries, err = transformer.Transform(queries)
```

Comment:
perhaps we could leverage `multierror` in this case instead of just returning first one?

Reply:
fixed